pkg/bucketeer/api/client.go

```diff
@@ -2,68 +2,40 @@
 package api
 
 import (
-	"context"
 	"fmt"
-
-	"google.golang.org/grpc"
-
-	protogateway "github.com/ca-dp/bucketeer-go-server-sdk/proto/gateway"
 )
 
 // Client is the client interface for the Bucketeer APIGateway service.
-type Client interface {
-	// GatewayClient defines GetEvaluation and RegisterEvents methods.
-	protogateway.GatewayClient
-
-	// Close tears down the connection.
-	Close() error
+type RestClient interface {
```

Comment:
Because we are not keeping both gRPC and rest implementations, it's unnecessary to rename using a prefix.

Reply:
talked at slack.